src/Zio/FileSystems/PhysicalFileSystem.cs

```diff
@@ -23,9 +23,9 @@ public class PhysicalFileSystem : FileSystem
     private const string DrivePrefixOnWindows = "/mnt/";
     private static readonly UPath PathDrivePrefixOnWindows = new UPath(DrivePrefixOnWindows);
 #if NETSTANDARD
-    private static readonly bool IsOnWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+    internal static readonly bool IsOnWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
 #else
-    private static readonly bool IsOnWindows = CheckIsOnWindows();
+    internal static readonly bool IsOnWindows = CheckIsOnWindows();
```

Comment:
Changed to internal for the OS-check in [MemoryFileSystem.cs](https://github.com/xoofx/zio/pull/87/files#diff-3100cc6fb29e710fbafa4f0cab7a2dae0998f0005393f5631ab9d820f2088b3c)